contrib/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Usage
+#
+#   cd contrib
+#   docker build -t pyatv-mqtt-bridge .
+#   docker run -d \
+#       --restart=always \
+#       --name=pyatv-mqtt-bridge \
+#       -v /opt/pyatv-mqtt-bridge/config.json:/app/config.json:ro \
+#       pyatv-mqtt-bridge
+#
+
+FROM ubuntu
```

Comment:
Have you tried whether you can also run it with an `alpine` image? In my case, the image ends up being 566 MB, which seems a bit large to me. I'll see if I can make it a little smaller in the next few days.

Reply:
Okay, I'll experiment a bit over the next few days.

Reply:
[Got it to work](https://github.com/sebbo2002/pyatv-mqtt-bridge/blob/feature/Dockerfile/Dockerfile) after some back and forth. The image has shrunk from 566MB to 165MB. It seems to work as far as it goes.